Dockerfile-CI.Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian:bullseye-slim as builder
+WORKDIR /builder
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    --no-install-recommends \
+    ca-certificates \
+    jq \
+    wget \
+    && case $(dpkg --print-architecture) in \
+      "amd64") \
+        wget -q -O - "$(wget -q -O- https://api.github.com/repos/lycheeverse/lychee/releases/latest \
+        | jq -r '.assets[].browser_download_url' \
+        | grep x86_64-unknown-linux-gnu)" | tar -xz lychee \
+      ;; \
+      "arm64") \
+        wget -q -O - "$(wget -q -O- https://api.github.com/repos/lycheeverse/lychee/releases/latest \
+        | jq -r '.assets[].browser_download_url' \
+        | grep  aarch64-unknown-linux-gnu)" | tar -xz lychee \
+      ;; \
```

Comment:
To make sure that the binaries exist we need to enable that block above, right?
```yaml
on:
  workflow_run:
     workflows: ["release"]
     types:
       - completed
       - requested
```
Because otherwise this workflow could finish before the binaries got created.